api/v1/ytsaurus_types.go

```diff
@@ -342,6 +348,42 @@ type DataNodesSpec struct {
 	Name string `json:"name,omitempty"`
 }
 
+type CRIJobEnvironmentSpec struct {
+	// Specifies wrapper for CRI service command.
+	//+optional
+	EntrypointWrapper []string `json:"entrypointWrapper,omitempty"`
+	// Sandbox (pause) image.
+	//+optional
+	SandboxImage *string `json:"sandboxImage,omitempty"`
+	// Timeout for retrying CRI API calls.
+	//+optional
+	RetryTimeoutSeconds *int32 `json:"retryTimeoutSeconds,omitempty"`
+	// CRI namespace for jobs.
+	//+optional
+	Namespace *string `json:"namespace,omitempty"`
+	// Base cgroup for jobs.
+	//+optional
+	BaseCgroup *string `json:"baseCgroup,omitempty"`
+}
+
+type JobEnvironmentSpec struct {
+	// Isolate job execution environment from exec node or not, by default true when possible.
```

Comment:
Maybe separate?

Reply:
No, i like "isolation" here.

"Isolate" here means jobs are isolated from the exec node while node stays where it is. 
There are strong subordination between components, "separate" is something about dividing comparable entities.